si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+        class `Error`:
+            {}: false
+        def `__init__`:
+            n/a: /
+            autocommit: false
+            Commit: Potrdi
+            Options: Možnosti
+            Topic evaluation: Vrednotenje teme
+            'Log perplexity: %(perplexity)s': Logaritemska perpleksnost: %(perplexity)s
+            'Topic coherence: %(coherence)s': Koherentnost teme: %(coherence)s
+        def `on_done`:
+            Latent Dirichlet Allocation: Latentna Dirichletova porazdelitev (LDA)
+            {:.5f}: false
+            c_v: false
+        def `send_report`:
+            Topics: Teme
+    class `TopicViewerTreeWidgetItem`:
+        def `__init__`:
+            {:d}: false
+            ', ': false
+            _: false
+        def `_color`:
+            '#ff6600': false
+            '#00cc00': false
+            '<span style="color: {}">{}</span>': false
+    class `TopicViewer`:
+        Topic: Tema
+        Topic keywords: Ključne besede teme
+    __main__: false
+    deerwester: false
+widgets/owtweetprofiler.py:
+    class `OWTweetProfiler`:
+        Tweet Profiler: Profiliranje tvitov
+        "Detect Ekman's, Plutchik's or Profile of Mood States's ": 'Zaznavanje čustev v tvitih po Ekmanovi, Plutchikovi ali POMS-ovi (Profile of Mood States) '
+        emotions in tweets.: kategorizaciji.
+        icons/TweetProfiler.svg: false
+        tweet profiler, twitter: tweet profiler, twitter, tvit, zaznavanje, čustva
+        class `Inputs`:
+            Corpus: Korpus
+        class `Outputs`:
+            Corpus: Korpus
+        class `Error`:
+            'Our servers are not responding. ': 'Naši strežniki se ne odzivajo. '
+            Please try again later.: Prosimo, poskusite pozneje.
+            'Unknown error: {}': Neznana napaka: {}
+        def `__init__`:
+            auto_commit: false
+            Commit: Potrdi
+            Cancel: Prekliči
+        def `generate_grid_layout`:
+            Options: Možnosti
+            tweet_attr: false
+            Attribute:: Atribut:
+            model_name: false
+            Emotions:: Čustva:
+            output_mode: false
+            Output:: Izhod:
+        def `send_report`:
+            Attribute: Atribut
+            Emotions: Čustva
+            Output: Izhod
+    __main__: false
+    election-tweets-2016.tab: false
+widgets/owtwitter.py:
+    def `search`:
+        content: false
+        @: false
+    class `OWTwitter`:
+        Twitter: true
+        Load tweets from the Twitter API.: Nalaganje tvitov iz družbenega omrežja Twitter preko API ključa.
+        icons/Twitter.svg: false
+        twitter, tweet: twitter, tweet, tvit
+        class `Outputs`:
+            Corpus: Korpus
+        class `Info`:
+            Downloaded fewer tweets than requested, since not enough tweets or rate limit reached: Prenesenih je bilo manj tvitov, kot je bilo zahtevanih, ker jih ni bilo dovolj ali ker je bila dosežena omejitev hitrosti.
```

Comment:
```suggestion
            Downloaded fewer tweets than requested, since not enough tweets or rate limit reached: Prenesenih je bilo manj tvitov, kot je bilo zahtevanih, ker jih ni bilo dovolj ali ker je bila dosežena omejitev dovoljenih prenosov iz APIja.
```